tools/api-markdown-documenter/src/test/test-data/simple-suite-test/test-suite-a.api.json

```diff
@@ -395,6 +395,101 @@
 					],
 					"implementsTokenRanges": []
 				},
+				{
```

Comment:
Adds a release level inheritance test case to the end-to-end test suite. Namespace is tagged as `@beta` and has a child export tagged with each release level.  The effective release level of each item should be the minimum of its tag and that of its ancestors (the namespace). I.e., `publicMember` will have an effective release level of `@beta` despite being tagged as `@public`.

The generated docs reflect this (I've left comments in some select places to help highlight the behavior).

Reply:
(Note that `@internal` items are unconditionally disregarded by API-Extractor, so there is no `internalMember` here.